tests/Issues/AddNodeBeforeNodeStmt/Source/AddBeforeStmtRector.php

```diff
@@ -34,17 +28,9 @@ public function getNodeTypes(): array
 
     public function refactor(Node $node)
     {
-        if (isset($this->justAdded[$this->file->getFilePath()])) {
-            return null;
-        }
-
-        $this->nodesToAddCollector->addNodeBeforeNode(
+        return [
```

Comment:
This test is on purpose for add node after node with inline html usages, so need to be reverted.

Reply:
It does the same, but now directly :+1:  The node adder will be removed, as not needed.